x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/time_comparison.spec.ts

```diff
@@ -51,32 +51,32 @@ const apisToIntercept = [
 ];
 
 describe.skip('Service overview: Time Comparison', () => {
-  before(async () => {
-    await synthtrace.index(
+  before(() => {
```

Comment:
Same here, are we waiting for the other PR's to run these tests?

Reply:
I'll leave it skipped for now. We should unskip in a follow-up PR.